pages/politicians/index.tsx

```diff
@@ -80,9 +83,22 @@ const PoliticianIndex: NextPageWithLayout = () => {
     (router.query.scope as PoliticalScope) || null
   );
   const [chamberFilter, setChamberFilter] = useState<Chambers | null>(
-    (router.query.chambers as Chambers) || null
+    (router.query.chamber as Chambers) || null
   );
 
+  useEffect(() => {
+    type RouteQuery = {
+      search: string | null,
+      scope: PoliticalScope | null,
+      chamber: Chambers | null 
+    }
+    let query: Partial<RouteQuery> = {}
+    if (!!debouncedSearchQuery) query.search = debouncedSearchQuery
+    if (!!politicalScope) query.scope = politicalScope
+    if (!!chamberFilter) query.chamber = chamberFilter
+    router.push({ query })
+  }, [debouncedSearchQuery, politicalScope, chamberFilter])
+
```

Comment:
I pushed this up a few weeks back and only fixed the build yesterday. Serves me right for not sanity checking it before PRing - taking a 👀

Reply:
no sweat! i'm not sure about this, but we may want to move this logic to the next server side in a `getServerSideProps` so that the query params are precomputed and passed as props _before_ render.

Reply:
This has been implemented.